contracts/Core/BlockManager.sol

```diff
@@ -138,18 +138,6 @@ contract BlockManager is Initializable, BlockStorage, StateManager, BlockManager
         _confirmBlock(epoch - 1, deactivatedCollections, stakerId);
     }
 
-    function disputeBiggestInfluenceProposed(
```

Comment:
why is this removed?

Reply:
a block with incorrect biggestInfluence can not be a part of the sortedProposedBlockIds. The only way i guess such a block can be a part of this is if all the block proposed have been incorrect.

Reply:
thats exactly why this was added.

Reply:
Added back https://github.com/razor-network/contracts/pull/602/commits/a1be8c7816f8525687ebcdd1da058b39c34158b9